src/features/rewards/welcomePage/style.ts

```diff
@@ -0,0 +1,174 @@
+
+import styled from 'styled-components'
+
+const centerBackground = require('./assets/img/centerTextBackground.svg')
+
+const StyledOptInSection = styled.section`
+  margin: 40px 0px 40px 0px;
+` as any
+
+const StyledOptInInnerSection = styled.section`
+  text-align: center;
+` as any
+
+const StyledCenterSection = styled.section`
+  padding: 15px 0px 40px 0px;
+` as any
+
+const StyledCenterContent = styled.div`
+  margin: 0 auto;
+  max-width: 675px;
+  padding: 40px 0px;
+` as any
+
+const StyledSection = styled.div`
+  display: block;
+` as any
+
+const StyledInfoContent = styled.section`
+  margin: 20px auto 0px auto;
+  max-width: 1000px;
+` as any
+
+const StyledTakeActionContent = styled.section`
+  margin: 80px auto 50px auto;
+  max-width: 500px;
+` as any
+
+const StyledBackground = styled.div`
+  background: url(${centerBackground}) no-repeat top;
+` as any
+
+const StyledFigure = styled.figure`
+  margin: 0 auto;
+  max-width: 115px;
+` as any
+
+const StyledBatLogo = styled.img`
+  margin: 20px auto 10px auto;
+  width: 115px;
+` as any
+
+const StyledDownArrow = styled.img`
+  margin: 0 auto;
+  color: #999;
+  width: 45px;
+` as any
+
+const StyledH1 = styled.h1`
+  font-size: 38px;
+  font-weight: 500;
+  color: #FFF;
+  letter-spacing: 0.16px;
+  display: inline-block;
+  margin: 14px 0px 10px 0px;
+` as any
+
+const StyledH2 = styled.h2`
+  color: #5C58C2;
+  font-size: 22px;
+  font-weight: normal;
+  letter-spacing: 0.16px;
+  line-height: 28px;
+` as any
+
+const StyledCenterTitle = styled.h2`
+  color: #222326;
+  font-size: 28px;
+  font-weight: normal;
+  text-align: left;
+  letter-spacing: 0.2px;
+  line-height: 28px;
+` as any
+
+const StyledSubTitle = styled.h2`
+  color: #5BC4FE;
+  font-size: 22px;
+  font-weight: 500;
+  text-align: center;
+  letter-spacing: 0.16px;
+  line-height: 28px;
+` as any
+
+const StyledTrademark = styled.span`
+  display: inline-block;
+  vertical-align: text-top;
+  margin-top: -25px;
+  color: #FFF;
+  font-size: 14px;
+  font-weight: 300;
+  opacity: 0.7;
+` as any
+
+const StyledRewardsParagraph = styled.p`
+  font-size: 16px;
+  max-width: 375px;
+  margin: 0 auto;
+  line-height: 28px;
+  color: #FFF;
+` as any
+
+const StyledTeaserParagraph = styled.p`
+  font-size: 16px;
+  margin: 0px 0px 10px 0px;
+  line-height: 28px;
+  color: #FFF;
+  opacity: 0.5;
+` as any
+
+const StyledReadyParagraph = styled.p`
+  font-size: 16px;
+  line-height: 28px;
+  color: #484B4E;
+  font-weight: 300;
+  letter-spacing: 0.16px;
+` as any
+
+const StyledCenterParagraph = styled.p`
+  font-size: 16px;
+  line-height: 28px;
+  color: #484B4E;
+  font-weight: 300;
+  letter-spacing: 0.16px;
+  max-width: 475px;
+  margin: 0 0 10px;
+` as any
+
+const StyledBoldParagraph = styled.p`
+  font-weight: 600;
+  font-size: 16px;
+  color: #484B4E;
+  margin: 0px 0px 60px 0px;
+` as any
+
+const StyledStrong = styled.strong`
+  font-weight: 600;
+` as any
+
+export {
```

Comment:
please do export on every line, so that we don't have this one big export at the end